src/__test__/ext.spec.js

```diff
@@ -171,32 +170,32 @@ describe('ext', () => {
       index = 2;
 
       indexRemoved(array, index);
-      expect(array).to.eql([0, 1]);
+      expect(array).toEqual([0, 1]);
     });
 
     it('should remove index the array and decrement', () => {
       array = [1, 2, 0, 3];
       index = 2;
 
       indexRemoved(array, index);
-      expect(array).to.eql([1, 0, 2]);
+      expect(array).toEqual([1, 0, 2]);
     });
   });
 
   describe('min', () => {
     it('should return 0 when > 0', () => {
-      expect(min(3)).to.equal(0);
+      expect(min(3)).toEqual(0);
```

Comment:
As I understand it:
- `to.equal` -> `toBe` (checking `===`)
- `to.eql`-> `toEqual` (checking deep equality)

So unless you are actually setting it to the right assertion, they should mapped to the correct assertion

Reply:
Good. Updated